src/lib/storage/vector_compression/bitpacking/bitpacking_vector.cpp

```diff
@@ -11,6 +11,10 @@ const pmr_compact_vector& BitPackingVector::data() const {
   return _data;
 }
 
+void* BitPackingVector::data_pointer() const {
```

Comment:
I don't see a usage of that method. Is it a left-over?

Reply:
I think it is supposed to be the `std::span` alternative to the `.data()` method of the `BitPackingVector`.
I'll remove it for now until it is needed.

Reply:
This was a left-over from first try outs of how to access typed information of underlying attribute vectors.